### DISCUSSION TITLE
Homebrew not installed in default location

### DISCUSSION THREAD

**Discussion by MrSebBird** [buresdv/Cork#175] [Q&A] [answered]
I have homebrew installed outside of its traditional path, would this be something that causes cork not to function?

It is not defecting any of the installed casks etc.

Am i able to change corks default path to find brew

Cheers

**Comment by buresdv** [comment, marked as answer]
Hey, this is not possible atm, but it will be added eventually. I have no ETA right now

**Comment by buresdv** [comment]
Implemented in the latest version
